docs: clarify removing an exporter via config requires an explicit actuator disable (#9204)
camunda/camunda-docs: docs/self-managed/components/orchestration-cluster/zeebe/operations/management-api.md

Reviewer: @eamonnmoloney can we put a note in https://github.com/camunda/camunda-docs/blob/e959728819346600a15ddb16518561669ea742fb/docs/self-managed/deployment/helm/configure/enable-additional-components.md?plain=1#L136 and link it to this page? Helm users would check that page
Also should we add this doc in Helm documentation https://artifacthub.io/packages/helm/camunda/camunda-platform#optimize-parameters?

Reviewer: This is addressed in https://github.com/camunda/camunda-docs/pull/9204/changes/6d67ae38e43e3dfebee8976f0a0c33d14d4a9723

Helm docs are handled in https://github.com/camunda/camunda-platform-helm/pull/6455

After disabling the exporter, no records will be exported to this exporter. Other exporters continue exporting.

> If you remove an exporter from the cluster configuration through Helm values — for example, by disabling Optimize, which removes the Elasticsearch or OpenSearch exporter — the exporter is only dropped from the static configuration, and the running exporter is not stopped automatically. The broker reconciles the removal into its dynamic cluster configuration only on partition leaders and relies on cluster gossip to converge across all brokers. If reconciliation does not converge, the exporter can remain active on some brokers, which prevents log compaction and increases disk usage. After removing an exporter through configuration, explicitly disable it using the request above, and confirm that every broker reports the exporter as `DISABLED` (see [Monitor an exporter](#monitor-an-exporter)).

Reviewer: I think we don't need to put lot of details

```suggestion
If you remove an exporter from the cluster configuration through Helm values — for example, by disabling Optimize, which removes the Elasticsearch or OpenSearch exporter — the exporter is only dropped from the static configuration, and the exporter is still declared in the dynamic cluster configuration, which prevents log compaction and increases disk usage. After removing an exporter through configuration, you need to explicitly disable it using the request above, and confirm that every broker reports the exporter as `DISABLED` (see [Monitor an exporter](#monitor-an-exporter)).
```
